src/scenes/Root/CommentsBar/CommentReply/CommentReply.tsx

```diff
@@ -0,0 +1,176 @@
+import { useMutation } from '@apollo/client';
+import loadable from '@loadable/component';
+import { Box, Button, TextField } from '@mui/material';
+import { EditorCore } from '@react-editor-js/core';
+import { useRef, useState } from 'react';
+import { StyleProps } from '~/common';
+import {
+  CreateOrReplyCommentDocument,
+  UpdateCommentDocument,
+} from './CommentReply.graphql';
+
+const EditorJsWrapper = loadable(() => import('~/components/EditorJsWrapper'), {
+  resolveComponent: (m) => m.EditorJsWrapper,
+});
+
+export interface CommentReplyProps extends StyleProps {
+  threadId?: string;
+  resourceId: string;
+  commentId?: string;
+  placeholder?: string;
+  blocks?: JSON;
+  isEditing?: boolean;
+  handleClose?: () => void;
+}
+
+export const CommentReply = ({
```

Comment:
I think this should be split. Editing a comment is a different use case than creating a new comment on a thread.

`EditComment` can receive a `commentId`.
`CommentReply` could receive a `parent` being a `ThreadFragment` or `CommentableFragment`.

This will help limit the prop surface area.